pronto/static/js/signatures/matrices.js

```diff
@@ -57,6 +57,37 @@ function getMatrices(accessions) {
 
             document.getElementById('overlaps').innerHTML = thead + `<tbody>${tbody1}</tbody>`;
             document.getElementById('collocations').innerHTML = thead + `<tbody>${tbody2}</tbody>`;
+
+            // Add exclusive-to-signature number of proteins
+            let exclusiveTableRows = ''
+
+            Object.entries(results.exclusive).forEach(([signature, count]) => {
+            const excludedSignatures = accessions.filter(acc => acc !== signature);
+            const linkToSignature =
+                count > 0
+                ? `<a href="/signatures/${signature}/proteins/?exclude=${excludedSignatures.join(',')}">${signature}</a>`
+                : sig;
```

Comment:
ReferenceError: `sig` is not defined

```suggestion
                : signature;
```